libcaja-private/org.mate.caja.gschema.xml

```diff
@@ -438,11 +438,21 @@
       <summary>Whether the navigation window should be maximized.</summary>
       <description>Whether the navigation window should be maximized by default.</description>
     </key>
+    <key name="extra-pane-position" type="i">
+      <default>-1</default>
+      <summary>Default position of the extra pane</summary>
+      <description>The default position (in pixels) of the extra pane in new windows, relative to the left side (or right side in right-to-left mode) of the navigation window.  A greater value here will allocate more space for the primary pane and less space for the extra pane.  A value of -1 here will split the window evenly in half between the two panes.</description>
+    </key>
```

Comment:
What happens if the user changes the screen resolution? What happens if the user has two screens with different resolutions? It may be better to use a percentage instead of an offset.